ignite/distributed/utils.py

```diff
@@ -496,7 +496,7 @@ def train_fn(local_rank, a, b, c):
     for comp_model_cls in registered_computation_models:
         if backend not in comp_model_cls.available_backends:
             continue
-        _set_model(comp_model_cls(backend, **kwargs))
+        _set_model(comp_model_cls(backend, **kwargs))  # type: ignore[arg-type]
```

Comment:
@vfdev-5 I had to set it to ignore here, because otherwise I get this error
```
error: Argument 1 to "_HorovodDistModel" has incompatible type "str"; expected "bool"
```
it also makes sense, because the `__init__` method is not defined as in the other models.

Reply:
Yes, this is weird actually in the sense how it worked previously with `_HorovodDistModel` :)

Reply:
Should I add it to the pile of shame? #1405 😄

Reply:
Never mind, I see, you created already a new issue for it 👍